docs/resources/docker.md.erb

```diff
@@ -25,8 +25,8 @@ or:
 
 where
 
-* `.where()` may specify a specific item and value, to which the matchers are compared
-* `commands`, `ids`, `images`, `labels`, `local_volumes`, `mounts`, `names`, `networks`, `ports`, `sizes`  and `'status'` are valid matchers for `containers`
+* `.where()` may specify a specific item and value, to which the resource parameters are compared
+* `commands`, `ids`, `images`, `labels`, `local_volumes`, `mounts`, `names`, `networks`, `ports`, `sizes`  and `'status'` are valid parameters for `containers`
```

Comment:
Could we remove one of the spaces after `sizes` there are two.

Reply:
done